R/mapGriddedData.r

```diff
@@ -299,8 +299,12 @@ mapGriddedData <- function(
 
     } #end of mapGriddedData()
 
+#' @importFrom raster raster

```

Comment:
@rsbivand just a tiny note: if you use `::` inside the function, e.g., `terra::rast` you do not need to add `#' @importFrom terra rast`. Roxygen should catch it. 

On the other hand, if you just use `rast` inside the function, then you should add `#' @importFrom terra rast` before

Reply:
I checked, but it wasn't caught, NAMESPACE lost the affected importFrom lines.